docs/features.md

```diff
@@ -0,0 +1,311 @@
+     },
+     {
+       "label": "Kiali",
+       "path": "apis/networking.istio.io/v1beta1/namespaces/kyma-system/virtualservices/kiali"
+     },
+     {
+       "label": "Tracing",
+       "path": "apis/networking.istio.io/v1beta1/namespaces/kyma-system/virtualservices/tracing"
+     }
+   ]
+ }
+}
+```
+
+- **PROTECTED_RESOURCES** – is used to block the edit and delete functions based on the determined rules. If the resource meets the rule requirements, the resource becomes protected and cannot be edited/deleted.
+  Each resource must have a match field, which is the list of key-value pairs. The proper rule description is when the definition given in the key matches the value.
+
+  Optionally you can provide a message parameter, which displays a simple message, or messageSrc, which is a yaml path where the message to be displayed is included. If neither message nor messageSrc is provided, a generic message will be used.
+
+Example:
+
+```bash
+"PROTECTED_RESOURCES": {
+  "isEnabled":true,
+  "config": {
+    "resources": [
+      {
+        # matches a resource with label "foo" equal to "bar"
+        "match": {
+          "$.metadata.labels.foo": "bar"
+        },
+        # message to display in the tooltip
+        "message": "This resource is protected"
+      },
+      {
+        # matches a resource with both label "foo" equal to "bar" and label "baz" equal to "qux"
+        "match": {
+          "$.metadata.labels.foo": "bar",
+          "$.metadata.labels.baz": "qux"
+        },
+        # source of the message to display in the tooltip
+        "messageSrc": "$.metadata.annotations.protected-message"
+      },
+      {
+        "match": {
+          "$.metadata.labels.foo": "bar"
+        }
+        # if neither "message" or "messageSrc" is given, a generic message will be used (from i18n files - key: common.protected-resource)
+      }
+    ]
+  }
+}
+```
+
+The match keys and messageSrc must use format as described at
+<https://github.com/dchester/jsonpath>.
+
+- **SENTRY** – is used to enable monitoring of uncaught exceptions, which then are analyzed and repaired. The address to which you send the information is located under the dsn key.
+
+Default settings:
+
+```bash
+"SENTRY ":{
+  "isEnabled": false,
+  "selectors":[ ],
+  "config": {
+    "dsn":""
+  }
+},
+```
+
+- **SERVERLESS** – is used to show or hide the Serverless view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
+  For the view to be shown, the feature must be enabled and all CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"SERVERLESS ":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"serverless.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **SERVICE_CATALOG** – is used to show or hide the Service Catalog view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
```

Comment:
**SERVICE_CATALOG** – is used to show or hide the Service Catalog views (Catalog, Instances and Brokers) 

SERVICE_CATALOG and SERVICE_CATALOG_ADDONS are also used to determine it the following components should be displayed.

FUNCTIONS_VIEW:
Service Bindings (in Configuration Tab)
Environment variables injected by the Service Bindings (in Code Tab)